sonic_installer/common.py

```diff
@@ -0,0 +1,27 @@
+
+import signal
```

Comment:
Seems this file is better fit into bootloader directory.

Reply:
Sure I can move it to the file that needs it.
It felt like a useful helper to have around and in `common.py` but if you prefer I can move it to the appropriate file now and once someone needs it somewhere else they'll move it back.

Reply:
Please help move it. I see so many reference to this file from bootloader.

---
In reply to: [443153279](https://github.com/Azure/sonic-utilities/pull/953#discussion_r443153279) [](ancestors = 443153279)

Reply:
Please be more argumentative about what problem you want me to solve here and how.
I fail to see the logic of your comment since main.py and all bootloader files need access to the variables/functions exposed by common.py
Moving common.py to bootloader would make no sense to me since main.py should not grab the `run_command` from bootloader since it doesn't have anything to do with what a bootloader is.
It is a utility module and to me lies at the perfect place.